server/events/runtime/run_step_runner_test.go

```diff
@@ -13,9 +13,10 @@ import (
 
 func TestRunStepRunner_Run(t *testing.T) {
 	cases := []struct {
-		Command string
-		ExpOut  string
-		ExpErr  string
+		Command     string
+		ProjectName string
```

Comment:
@nikovirtala what I intended was you add the `ProjectName` here

Reply:
I see. You give the ProjectName a value inside the cases struct. - That's what I was wondering how to do it. As easy as its part. :)